349/step3.py

```diff
@@ -0,0 +1,20 @@
+from typing import List
+
+
+class Solution:
+    def intersection(self, nums1: List[int], nums2: List[int]) -> List[int]:
+        if len(nums1) > len(nums2):
+            long_nums = nums1
+            short_nums = nums2
+        else:
+            long_nums = nums2
+            short_nums = nums1
```

Comment:
```suggestion
        short_nums, long_nums = sorted([nums1,nums2], key=len)
```
サイズの大小で変数に振り分けているコードについて、こちらのような書き方もあるようです。
個人的にはこちらの書き方の方がシンプルで良いなと思いました。

Reply:
ご指摘ありがとうございます、このような書き方ができるのを初めて知りました。おっしゃるようにすっきり書けますね。勉強になります。